components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/handler/request/impl/DefaultAuthenticationRequestHandler.java

```diff
@@ -387,6 +391,42 @@ protected void concludeFlow(HttpServletRequest request, HttpServletResponse resp
                 }
             }
 
+            // Break the flow if the authenticated subject identifier or user id is null.
+            if (StringUtils.isBlank(sequenceConfig.getAuthenticatedUser().getAuthenticatedSubjectIdentifier())) {
+                if (LoggerUtils.isDiagnosticLogsEnabled()) {
+                    LoggerUtils.triggerDiagnosticLogEvent(
+                            FrameworkConstants.LogConstants.AUTHENTICATION_FRAMEWORK, null, LogConstants.FAILED,
```

Comment:
Can't we add the application name, and username kind of details for the second parameter in the triggerDiagnosticLogEvent